front/src/components/addSurvey.js

```diff
@@ -0,0 +1,498 @@
+import React, {Component} from "react";
+import "../App.css";
+import '../styles/survey.css';
+import "survey-react/survey.css";
+import * as Survey from "survey-react";
+import axios from 'axios';
+
+class AddSurvey extends Component {
+
+    constructor(props) {
+        super(props);
+
+
+        this.state = {
+            id: 0,
+            inputValue1: '',
+            inputValue2: '',
+            inputValue3: '',
+            inputValue4: '',
+            inputValue5: '',
+            isRequired: false,
+            tmpQuestion: {"questions": []},
+            frontUrl: "http://localhost:3000",
+            serverUrl: "http://localhost:8080",
+            templateId: "",
+            current_survey: {"questions": []},
+            testSurvey: {
```

Comment:
don't hardcode test survey. it can be in some other place (another file). Or tests can be written 😛